homework_2/task3.js

```diff
@@ -0,0 +1,15 @@
+const checkLastNumber = (a, b) => {
+    let digit1 = a % 10;
+    let digit2 = b % 10;
+    if (
+        typeof digit1 === "number" && 
```

Comment:
a, b should be the number, please remove 5 and 6 lines, and use these condition for a and b variables